app/src/main/java/com/franckrj/respawnirc/WebBrowserActivity.java

```diff
@@ -152,6 +152,10 @@ public boolean onOptionsItemSelected(MenuItem item) {
                 return true;
             case R.id.action_reload_page_webbrowser:
                 browserWebView.reload();
+                return super.onOptionsItemSelected(item);
```

Comment:
Le fall-though de `case R.id.action_reload_page_webbrowser` était plus un bug qu'autre chose je dirais, ça devrait retourner `true` normalement.

```suggestion
                return true;
```

Reply:
c'est réglé